python/whylogs/viz/notebook_profile_viz.py

```diff
@@ -275,18 +241,19 @@ def summary_drift_report(self, cell_height: str = None) -> HTML:
             visualization.summary_drift_report()
 
         """
+        if not self._target_view and not self._ref_view:
```

Comment:
Should this be `or` instead of `and` ?  `if not (self._target_view and self._ref_view):` would match the error message.